components/folder/FolderList.tsx

```diff
@@ -0,0 +1,55 @@
+import Link from 'next/link';
+
+import { getFolderList } from '@/apis/api';
+
+const TOTAL_LIST_NAME = "전체"
+
+interface DataItem {
+  id: number;
+  name: string;
+}
+
+const getFolderListData = async () => {
+  try {
+    const { data } = await getFolderList();
+
+    return data;
+  } catch {
+    throw new Error('error');
+  }
+}
+
+const FolderList = async () => { // 수정 부분
+  const data = await getFolderListData();
```

Comment:
```suggestion
  const folderList = await getFolderListData();
```

data보다는 명시적으로 어떤 역할을 수행하는 변수인지 명시하면 좋습니다. 타입도 제공되지 않는 상황에서는 변수명으로만 값을 유추할 수 없기 때문에 권장드립니다.